rllib/tuned_examples/bc/benchmark_atari_pong_bc.py

```diff
@@ -145,31 +146,32 @@ def _env_creator(cfg):
 # There are many run numbers, we choose the first one for demonstration. This
 # can be chosen by users. To use all data use a list of file paths (see
 # `num_shards`) and its usage further below.
-run_number = 1
-# num_shards = 1
+# run_number = 1
```

Comment:
Should we just remove these? Or do you think they are still helpful for users?

Reply:
Yeah, we could actually use a run-number or use all of them. It just makes clear how the data is structured - for execution it is not needed. Also the code below downloading the data to the disk is not anymore needed as soon as downloading directly from the bucket works on the workspace.